corehq/ex-submodules/pillowtop/models.py

```diff
@@ -76,3 +76,19 @@ class KafkaCheckpoint(models.Model):
 
     class Meta:
         unique_together = ('checkpoint_id', 'topic', 'partition')
+
+    @classmethod
+    def create_for_checkpoint_id(cls, checkpoint_id, topics):
+        tps = list(
+            cls.objects
+            .filter(topic__in=topics)
+            .distinct('topic', 'partition')
```

Comment:
what happens when we alter the partitions? guessing it shouldn't affect anything, but wonder if there should be any cleanup or if there are any edge cases